### ISSUE TITLE
does libp2p have gossiping related feature?

### ISSUE THREAD

**Issue by lzl124631x** [libp2p/go-libp2p#514]
I know libp2p has relay but that is one peer to another. Does libp2p have gossiping feature with which one peer can broadcast message to the whole network? (Consider bitcoin transaction gossiping)

**Comment by vyzo** [comment]
yes, we have [pubsub](https://github.com/libp2p/go-libp2p-pubsub).
